Library/SharedFunctions.swift

```diff
@@ -106,7 +111,9 @@ internal func minAndMaxPledgeAmount(forProject project: Project, reward: Reward?
   case .none, .some(Reward.noReward):
     return (Double(country.minPledge ?? 1), Double(country.maxPledge ?? 10_000))
   case let .some(reward):
-    return (reward.minimum, Double(country.maxPledge ?? 10_000))
+    /// We  want to use `reward.latePledgeAmount`  if the backing was't made in a late pledge state. Otherwise, we'll use `reward.pledgeAmount` if it was made in a normal crowdufunding state.
+    let amount = isLatePledgeBacking ? reward.latePledgeAmount : reward.pledgeAmount
+    return (amount, Double(country.maxPledge ?? 10_000))
```

Comment:
This is the piece that I mentioned being slightly confused by, but my understanding is that `reward.minimum` is whatever the current reward amount is. 

Not that we're choosing to use the amount based on the campaign state the pledge was in, I'm questioning if we need reward.minimum at all anymore.